packit_service/worker/handlers/bodhi.py

```diff
@@ -133,12 +135,22 @@ def run(self) -> TaskResults:
             issue_repo = self.service_config.get_project(
                 url=self.job_config.issue_repository
             )
-            body = (
-                f"Bodhi update failed to be created for `{self.koji_build_event.nvr}`:\n"
-                "```\n"
-                f"{ex}\n"
-                "```"
-            )
+
+            if isinstance(ex.__cause__, AuthError):
+                body = (
+                    f"Bodhi update failed to be created for `{self.koji_build_event.nvr}` "
+                    f"because of the missing permissions.\n\n"
+                    f"Please, give {self.service_config.fas_user} user `commit` rights in the "
+                    f"[dist-git settings]({self.data.project_url}/adduser)."
```

Comment:
Will this be enough? This is a genuine question since I'm lost here at this point :D

Reply:
It did the job for `python-requre` and `packit-stg` FAS. We can easily enhance this message once we know more...